py/scripts/start.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+echo "RUNNING SERVER"
+gunicorn -w 1 -b 0.0.0.0:4000 src.wsgi:app
```

Comment:
Todo joya lo que vi, una boludez nomás:

Lo único que me genera duda es porque estamos usando el 0.0.0.0 en vez del 127.0.0.1
Entiendo que ambos apuntan al localhost pero la diferencia es que el 127 hace uso de la capa de transporte (en este caso tcp) y el 0.0.0.0 bypassea todo eso (lo cual creo que no es lo ideal en el contexto de hacer requests, por más que sean mock).

Yo probaria cambiar esa linea por localhost:4000. En caso de que siga andando creo que sería lo ideal.
Leéte [esto](https://www.howtogeek.com/225487/what-is-the-difference-between-127.0.0.1-and-0.0.0.0/) si te pinta entender por qué te digo lo que te digo jajaj

Reply:
Ah genial! dale ahi lo leo y lo cambio :+1:

Reply:
Estuve probando y si escribo esto asi ( `localhost:4000.` ) no anda, como que no estuviera expuesta la direccion del host. Pero si escribo esto `gunicorn -w 1 -b :4000 src.wsgi:app` anda y lo sirve en http://127.0.0.1. No se que onda.

Reply:
Perfecto! joya, mejor que quede generico y que lo resuelva el SO :)
Mañana tengo tiempo armo el de `node` asi ya empezamos con pruebas. Ahora me estoy bajando el ultimo cap de Rick & Morty pero mañana me pongo jejej.

Reply:
JAJAJAJ tranqui no hay drama. Yo iba a tirar pr ahora de lo de node ya. asi me lo sacaba de encima

Reply:
Be my guest

Reply:
jaja dale :+1: